bubble.rb

```diff
@@ -0,0 +1,19 @@
+def bubble_sort_by(arr)
+  b = false
+  while b
+    b = false
+    a.length.times do |i|
+      if arr[i] != arr.last && a[i] > a[i + 1]
```

Comment:
- Be consistent in the name of the array variable you are working. 'arr' or 'a'?